chartpress.py

```diff
@@ -295,9 +299,12 @@ def build_values(name, values_mods):
                 )
 
             mod_obj['tag'] = value['tag']
+        elif isinstance(mod_obj, str) and set(value.keys()) == {'repository', 'tag'}:
```

Comment:
the `build_images` function will return the `values_mods`, which will be used to get the `value` variable.

And this is how the `values_mods` are set:

```
        value_modifications[options['valuesPath']] = {
            'repository': image_name,
            'tag': SingleQuotedScalarString(image_tag),
        }
```

That makes me think that `set(value.keys()) == {'repository', 'tag'}` would always evaluate to true.

Reply:
indeed it will, for now! This was a protection against the fact that `values_mods` could be* a generic update to any fields in values, so may in the future contain updates other than image tags. But you are right that it does not right now, so this will always be true.

I updated the assumption to be explicit a little earlier, rather than checking it here, since I realized the dict-update also made the same assumption.